Formula/h2c.rb

```diff
@@ -0,0 +1,15 @@
+class H2c < Formula
+  desc "Headers 2 curl"
+  homepage "https://curl.se/h2c/"
+  url "https://github.com/curl/h2c/archive/fee2a0361b4124dbd5565cee2a90ffb0c80ab7b0.tar.gz"
+  sha256 "8eba84aef62506dfff4f0d975a062fa5f347c3298784c8f4f778a0244fec4df0"
+  license "MIT"
+
+  def install
+    bin.install "h2c"
+  end
+
+  test do
+    assert_match "h2c.pl [options] < file", shell_output("h2c --help")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.